TalismanSociety/signet-web
# [FEAT] - SplitAbstain vote & Remove Vote

# Description

## ⛑️ **What was done:**

- Added SplitAbstain vote functionality
- Display pending votes that are not yet concluded
- Change and Remove votes of ongoing referendums

## Type of change


### 🧪 **How to test:**
- SplitAbstain Vote functionality:

1. Visit `/voting`
2. Click on "Abstain"
3. Select a proposal
4. Fill in amounts to Abstain
5. Click review
6. Sign transaction with all required signers

Expected behavior: 
- Transaction should go through, transaction details should be displayed correctly on side sheet on every transaction status
- Transaction should be displayed on Pending Votes list on `/votes`
- Only the latest vote should be displayed on Pending votes list

- Remove Vote functionality:
1. Visit `/voting`
2. If the multisig doesn't have a Aye, Nay or Abstain transaction, follow the SplitAbstain Vote functionality steps above
3. Click "Remove" on Pending votes list
4. Sign transaction with required signers

Expected behaviour:
- Transaction should go through and displayed on `/overview`
- Vote should no longer be displayed on Pending votes list

### 🗒️ **Notes:**


### 🖼️ **Screenshots:**


https://github.com/TalismanSociety/signet-web/assets/47801291/7e4cce0b-5fe8-4d21-af35-040c1318fdbb
